.docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,7 @@
+version: '3.4'
```

Comment:
there are newer versions of this - originally this was related to Travis CI having a old version of `docker-compose` installed by default, is this still the case for Github Actions?

Reply:
The [ubuntu 18.04 runner](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) has [docker-compose 1.25.4](https://github.com/docker/compose/releases/tag/1.25.4). But i'm struggling to determine the docker-engine version. They only refer to docker-moby 3.0.11.

Reply:
This is the version of the config file format rather than the engine. We don't need any of the newer features yet, and isn't limited to just GitHub Actions.